docs/guides/vpn/wireguard/client.md

```diff
@@ -201,12 +202,13 @@ That's it.
 You can now copy the configuration file to your client (if you created the config on the server). If the client is a mobile device such as a phone, `qrencode` can be used to generate a scanable QR code:
 
 ```bash
-sudo qrencode -t ansiutf8 < "/etc/wireguard/${name}.conf"
+qrencode -t ansiutf8 < "/etc/wireguard/${name}.conf"
+exit
 ```
 
 (you may need to install `qrencode` using `sudo apt-get install qrencode`)
 
-You can directly scan this QR code with the official WireGuard app after clicking on the blue plus symbol in the lower right corner.
+You can directly scan this QR code with the official WireGuard app after clicking on the blue plus symbol in the lower right corner. You can also run `sudo qrencode -t ansiutf8 < "/etc/wireguard/[client_name].conf"` to display the QR code after exiting sudo mode.
```

Comment:
I'm not sure if this addition is needed. It's basically a repetition of two lines above with added `sudo`

Reply:
I'll try to make clearer. I was trying to get at that, once you've gone through this process with the saved client name, and exited the sudo mode, you can go back to the terminal at any time and get the QR code to display if you know the client name.